tests/helpers/acceptance.js

```diff
@@ -138,38 +77,33 @@ function teardownTestTargets() {
  * @return {Promise}
  */
 function linkDependencies(projectName) {
-  var targetPath = path.join(root, 'tmp', projectName);
-  return tmp.setup(targetPath).then(function() {
-    return copy(path.join(root, 'common-tmp', projectName), targetPath);
-  }).then(function() {
-    var nodeModulesPath = path.join(targetPath, 'node_modules');
+  var sourceFixture = dirs[projectName]; // original fixture for this acceptance test.
+  var runFixture = quickTemp.makeOrRemake(dirs, projectName + '-clone');
 
-    mvRm(nodeModulesPath, path.join(root, '.deps-tmp', 'node_modules'));
+  fs.copySync(sourceFixture, runFixture);
 
-    if (!existsSync(nodeModulesPath)) {
-      symLinkDir(targetPath, path.join(root, '.deps-tmp', 'node_modules'), 'node_modules');
-    }
+  var nodeManifest = fs.readFileSync(path.join(runFixture, 'package.json'));
 
-    process.chdir(targetPath);
+  var packageCache = new PackageCache({ linkEmberCLI: true });
+  packageCache.create('node', 'npm', nodeManifest);
 
-    var appsECLIPath = path.join('node_modules', 'ember-cli');
-    fs.removeSync(appsECLIPath);
+  var nodeModulesPath = path.join(runFixture, 'node_modules');
+  symlinkOrCopySync(path.join(packageCache.get('node'), 'node_modules'), nodeModulesPath);
 
-    // Need to junction on windows since we likely don't have persmission to symlink
-    // 3rd arg is ignored on systems other than windows
-    fs.symlinkSync(path.resolve('../..'), appsECLIPath, 'junction');
+  process.chdir(runFixture);
 
-    return targetPath;
-  });
+  return Promise.resolve(runFixture);
```

Comment:
Since I don't expect the future state version of this to be `async` I've moved away from promises. Keeping it as a promise was just a consequence of "it says it returns a promise!" and not thinking about it.